packages/devextreme/testing/tests/DevExpress.ui.widgets/tileView.tests.js

```diff
@@ -276,6 +276,28 @@ QUnit.module('rendering', {
         assert.strictEqual($tile.outerHeight(), DEFAULT_ITEMSIZE, 'Tile height updated correctly');
         assert.strictEqual($tile.outerWidth(), DEFAULT_ITEMSIZE, 'Tile width updated correctly');
     });
+
+    QUnit.test('aria-activedescendant should not be set for the component after tile focus (T1217255)', function(assert) {
```

Comment:
Should we use 'testInActiveWindow' instead of test?

Reply:
[b59a80dcb8b777fb151f57a2c9809f8aa4d10140]